### PR TITLE
w/signoff

### PR DESCRIPTION
**Pull request by remote-event**
Signed-off-by: Luna

### CODE REVIEW

**Comment by remote-event** [comment]
Fully tested to make sure everything is working. Please let me know if there are an issues that appear.

**Changes I've made...**
 - I noticed that you're using an Linux command to set the node environment variable. It won't work if you're using Windows , so I installed the cross-env module that set and use environment variables across platforms. 
- Documented the new commands ``multi-add`` and ``multi-remove`` in README.md and the list command.
- Refactored the code in ``index.js`` that loads each event. Events will have to all load before bringing the Discord bot online.
- Improved some commands. Adds the entire user + discriminator for the footer and the database who added the new element.

Also , I accidentally left my credentials in .example.env but they're all invalidated now.

File: src/config/.env.example [hunk 1]
```diff
@@ -1,7 +1,7 @@
 PREFIX=$
-BOT_TOKEN={TOKEN FOR YOUR DISCORD BOT}
-LOCAL_DB_USER={NAME_FOR_LOCAL_DB_USER}
-LOCAL_DB_PASSWORD={NAME_FOR_LOCAL_DB_PASSWORD}
-LOCAL_DB_MONGO={NAME_FOR_LOCAL_DB_MONGO}
+BOT_TOKEN=NzEwNzc3MzU1NzEwMzY1ODA3.Xr5Y_w.fzBW6TeXr1ABEGLBBtHXqItyBdU
+LOCAL_DB_USER=Luna
+LOCAL_DB_PASSWORD=PPvIA6LkJEdpqEIS
+LOCAL_DB_MONGO=storage
```
Comment (tomassirio) [inline src/config/.env.example]: @TheRealLunatite Watch out, you have your credentials here. Take them out

**Review by tomassirio — CHANGES_REQUESTED** [review]
I like the changes. However, check the issue with the .env file and then i'll merge changes

File: src/config/.env.example (hunk 1 above)
Comment (remote-event) [reply src/config/.env.example]: Hi! I made changes to my PR by removing the sensitive information out of the ``.example.env`` file. The credentials are invalidated anyhow.

**Pull request merged**